x/simulation/util.go

```diff
@@ -91,3 +76,12 @@ func PeriodicInvariants(invariants []sdk.Invariant, period int, offset int) []sd
 	}
 	return outInvariants
 }
+
+func mustMarshalJSONIndent(o interface{}) []byte {
+	bz, err := json.MarshalIndent(o, "", "  ")
+	if err != nil {
+		panic(fmt.Sprintf("failed to JSON encode: %s", err))
```

Comment:
```suggestion
		panic(fmt.Sprintf("failed to JSON encode: %s", err.Error()))
```

Reply:
This is redundant go will automatically call `String` when `%s` is used FYI ;-)